src/client/containers/Profile/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Profile() {
```

Comment:
Can we make an example of how to fx show the users email?

Reply:
It is not really related to Authentication but we can add it. I just want to avoid adding any state management in the boilerplate. If possible :)